src/client/java/teammates/client/scripts/DataMigrationForRubricQuestionsWithWeightsAttached.java

```diff
@@ -103,10 +103,11 @@ protected FeedbackQuestion updateWeights(FeedbackQuestion question) {
             rubricWeightsForEachCell.set(fqd, weightsForEachCell);
 
             attr.setQuestionDetails(fqd);
-            return attr.toEntity();
+            question.setQuestionText(attr.questionMetaData);
+            question.keepUpdateTimestamp = true;
         } catch (ReflectiveOperationException e) {
             System.out.println(e.getMessage());
-            return question;
         }
+        return question;
```

Comment:
It turns out that we don't even need to return the question.

Reply:
@xpdavid Is it because we are modifying the same question object?

Reply:
Yes. It is just a code quality issue.

Reply:
Okay. Removing it.